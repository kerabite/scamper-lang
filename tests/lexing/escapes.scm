(define str "ab\ncd\te\n\n\"\'zzz\
zzz\"")