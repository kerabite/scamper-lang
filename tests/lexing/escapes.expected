(: (1, 1) => (1, 1)
define: (1, 2) => (1, 7)
str: (1, 9) => (1, 11)
"ab
cd	e

"'zzzzzz"": (1, 13) => (2, 6)
): (2, 7) => (2, 7)
